### ISSUE TITLE
runServer Cannot find module '......\build\[name].js'

### ISSUE THREAD

**Issue by neXtack** [kriasoft/react-starter-kit#1372]
Clone the latest code, `npm i`, `npm run build`, and then `npm run serve`, No suprise, but an error occured.
- branch : 
master

- error :
When `npm run serve`, an error occurred.
``` javascript
Error: Cannot find module 'E:\opensource\react-starter-kit\build\[name].js'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3
```

- cause :

The `serverPath` is not a real path, but a filename template containing `[name]`。

1. tools/webpack.config.js
serverConfig section
```javascript
...
output: {
    ...config.output,
    path: path.resolve(__dirname, '../build'),
    filename: '[name].js',
    chunkFilename: 'chunks/[name].js',
    libraryTarget: 'commonjs2',
},
...
```
2. tools/runServer.js
```javascript
const [, serverConfig] = webpackConfig;
const serverPath = path.join(
  serverConfig.output.path,
  serverConfig.output.filename,
); // serverPath = 'build/[name].js'
```

**Comment by frenzzy** [comment]
if you want to use `npm run serve` then you must compile app first via `npm run build`, but it is easier just use `npm start` for development or `npm run build -- --release && node build/server.js` for production, see also #1275

**Comment by Toosick** [comment]
Yeah I'm trying to run `yarn run deploy` and it doesn't work. Changing
```
const serverPath = path.join(
  serverConfig.output.path,
  serverConfig.output.filename,
);
```
to
```
const serverPath = path.join(
  serverConfig.output.path,
  'server.js',
);
```
worked.

**Comment by frenzzy** [comment]
Sorry, totally forgot about updating this script. Fixed 9aafdb4

**Comment by Toosick** [comment]
I also had to do `publicPath: isDebug ? '/assets/' : 'assets/',` in the webpack config so GH pages would work.
